cadc-registry/src/main/java/ca/nrc/cadc/reg/XMLConstants.java

```diff
@@ -149,5 +151,10 @@ public class XMLConstants {
 
     public static final Namespace CAPABILITIES_NS = Namespace.getNamespace("vosi", VOSICAPABILITIES_10_NS.toString());
     public static final Namespace VODATASERVICE_NS = Namespace.getNamespace("vod", VODATASERVICE_11_NS.toString());
+    public static final Namespace VODATASERVICE_VS_NS = Namespace.getNamespace("vs", VODATASERVICE_11_NS.toString());
+    public static final Namespace VORESOURCE_NS = Namespace.getNamespace("vr", VORESOURCE_10_NS.toString());
 
+    public static final List<Namespace> NAMESPACE_LIST =
```

Comment:
This NAMESPACE_LIST constant seems specific to a capabilities document so it probably belongs in CapabilitiesWriter directly. It is certainly a subset of all the NS constants declared here and included in the SCHEMA_MAP.

yeah, move to  CapabilitiesWriter

Reply:
Done.